src/Tools/DynamoInstallDetective/Utilities.cs

```diff
@@ -61,10 +61,32 @@ public static IEnumerable LocateDynamoInstallations(string additionalDynamoPath,
         /// version info as Tuple of the file found in the installation based 
         /// on file search pattern. The returned list is sorted based on version 
         /// info.</returns>
+        [Obsolete("Function will be deprecated in Dynamo 3.0, please use FindProductInstallations2")]
```

Comment:
this message seems a bit odd to me - in Dynamo 3.0 we can just use this method again - now we have to obsolete this method and use the new one.

Reply:
If no objection, I will change to say the function signature will change in Dynamo 3.0?